packages/utils/node-resolver-core/src/NodeResolver.js

```diff
@@ -1149,8 +1146,6 @@ export default class NodeResolver {
           sourcePath: pkg.pkgfile,
           resolved: alias.global,
         };
-      } else if (alias.fileName) {
-        alias = alias.fileName;
```

Comment:
Flow said that this is unreachable type-wise. And this branch was indeed never reached in the tests. So either it really is dead, or we don't test this at all.